shuffle/cli/src/account.rs

```diff
@@ -15,15 +15,42 @@ use diem_types::{
 };
 use generate_key::load_key;
 use shuffle_transaction_builder::framework::encode_create_parent_vasp_account_script_function;
-use std::path::PathBuf;
+use std::fs;
+use std::io::Write;
+use std::os::unix::fs::PermissionsExt;
+use std::path::Path;
+use std::time::SystemTime;
+use structopt::StructOpt;
 
-pub fn handle(project_dir: PathBuf, account_key_path: PathBuf) -> Result<()> {
-    let config_path = project_dir.join("nodeconfig/0").join("node.yaml");
+// Creates new account from randomly generated private/public key pair.
+pub fn handle() -> Result<()> {
+    let node_config_path = &get_shuffle_dir();
+    if !Path::new(node_config_path.as_path()).is_dir() {
+        println!("A node hasn't been created yet! Run shuffle node first");
+        return Ok(());
+    }
+    let dev_keys_dir = &node_config_path.join("keys");
+    let address_dir = &node_config_path.join("addresses");
+
+    if !Path::new(dev_keys_dir.as_path()).is_dir() && !Path::new(address_dir.as_path()).is_dir() {
+        fs::create_dir(dev_keys_dir)?;
+        fs::create_dir(address_dir)?;
+        fs::set_permissions(dev_keys_dir, fs::Permissions::from_mode(0o700))?;
+        fs::set_permissions(address_dir, fs::Permissions::from_mode(0o700))?;
```

Comment:
what if the dir exists but the permissions aren't right?

Reply:
Discussed offline. Removing permissions